corehq/apps/reports/export.py

```diff
@@ -89,7 +90,7 @@ def generate_bulk_file(self):
                                 separator=self.separator)
                         ])]
                     )
-
+                update_progress(i+1)
```

Comment:
It looks like this will be called once per config in the bulk export, not once per document. It would be better to put inside `for doc in config.get_docs()` I think.